### PR TITLE
NEW: NotificationBadge

### PR DESCRIPTION
**Pull request by tomashapl**
<br/><br/><br/><url>LiveURL: https://orbit-components-notification-badge.surge.sh</url>

### CODE REVIEW

**Comment by filipdanisko** [comment]
@honzatmn @tomashapl Please guys, how this is going to be used? 

I think it would be good idea to support aria tags to reflect notification behaviour. Minimum I would say would be to support aria-label, but more advanced solution for inspiration is [here.](https://davidetriso.github.io/aria-badge/) Containing `aria-live`, `aria-atomic` and `role`

File: src/NotificationBadge/index.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+StyledNotificationBadge.defaultProps = {
+  theme: defaultTheme,
+};
+
+const NotificationBadge = (props: Props) => {
+  const { type = TYPE_OPTIONS.NEUTRAL, children, icon, ariaLabel, dataTest } = props;
+
+  return (
+    <StyledNotificationBadge>
+      <Badge type={type} dataTest={dataTest} icon={icon} ariaLabel={ariaLabel}>
```
Comment (vepor) [inline src/NotificationBadge/index.js]: Should we support `icon` for NotificationBadge? @honzatmn

File: src/NotificationBadge/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Badge
```
Comment (vepor) [inline src/NotificationBadge/README.md]: NotificationBadge in the entire file 🙏

Comment (tomashapl) [reply src/NotificationBadge/README.md]: nice catch 👏 updated

**Pull request merged**